src/uct/cuda/gdr_copy/gdr_copy_iface.c

```diff
@@ -46,6 +46,10 @@ static ucs_config_field_t uct_gdr_copy_iface_config_table[] = {
          ucs_offsetof(uct_gdr_copy_iface_config_t, put_latency)},
         {NULL})},
 
+    {"RCACHE_OVERHEAD", "250ns",
```

Comment:
I think it should be in `uct_gdr_copy_md_config_table` right after `UCX_GDR_COPY_RCACHE`.
And, perhaps `md->reg_cost` should be initialized using the overhead, if rcache is enabled.

Reply:
Agree with you. We discussed that with @brminich and came to a conclusion that it should be different `md_attr` field similar to `reg_cost` but `lookup_cost`. Also it seems like `ucp_proto_init_add_memreg_time` doesn't differentiate cacheable and non-cacheable MDs and adds UCP rcache overhead even for GDRcopy which is wrong. Also we need to somehow consider that overhead separately for `memtype_ep` operation if they used to do D2H and H2D copies which extends the work that need to be done significantly.